package.json

```diff
@@ -189,6 +189,14 @@
           "testing-library/render-result-naming-convention": "off"
         }
       },
+      {
+        "files":  ["packages/redux/**/*.ts?(x)", "packages/redux/**/*.spec.ts?(x)"],
```

Comment:
can you also add a `yarn test:redux` entry?

Reply:
Good idea. Unfortunately `packages/redux/src/createSliceHooks/index-integration.spec.tsx` seems to be failing right now

Reply:
Bummer. Puppeteer config issue?

Reply:
it's not a e2e test. The name is misleading but it's just an RTL test. I cannot rename otherwise updating the package will become an issue over time.